mne/io/snirf/tests/test_snirf.py

```diff
@@ -42,6 +43,10 @@
 nirx_nirsport2_20219 = op.join(data_path(download=False), 'NIRx',
                                'nirsport_v2', 'aurora_2021_9')
 
+# Kernel
+kernel_hb = op.join(data_path(download=False), 'SNIRF', 'Kernel', 'Flow50',
```

Comment:
At some point we should really refactor this to be testing_path = data_path(...) so that the data path calls can be more DRY

Reply:
Done